### ISSUE TITLE
Build documentation 

### ISSUE THREAD

**Issue by SaashaJoshi** [SaashaJoshi/piQture#116]

**Comment by JagjeetChauhan** [comment]
I would like to contribute, under Hacktoberfest.

**Comment by SaashaJoshi** [comment]
Hi @JagjeetChauhan, this is great. The documentation can be divided into several parts. Would you be interested in overall improving the documentation page, [here](https://saashajoshi.github.io/piQture/), or in fact in building some demo or tutorial?

**Comment by JagjeetChauhan** [comment]
I would like to improve the documentation page on the website and then I will try to add some demo's and tutorials.

**Comment by SaashaJoshi** [comment]
That is awesome. Could you open a secondary issue titled something like "Improve documentation page" and I will tag it for hacktoberfest. This will help me to focus on a particular thing. Tag me anytime you require any help!

**Comment by JagjeetChauhan** [comment]
ok Sure.

**Comment by Dljdd** [comment]
Hi I can also help with the documentation part of the codebase

**Comment by SaashaJoshi** [comment]
Hi @Dljdd, 
Do you have anything particular in mind? For example, would you be interested in improving the documentation pages or perhaps add some demos or tutorials?

**Comment by Dljdd** [comment]
I would be interested in adding demos/tutorials to the codebase

**Comment by SaashaJoshi** [comment]
Awesome! Would you prefer to do that for #131?

**Comment by Dljdd** [comment]
Yes I'd also try adding it for modules that currently don't have one and need it, if you have any such modules in mind then do let me know!
